src/Ledger/Ratify.lagda

```diff
@@ -294,5 +533,24 @@ data _⊢_⇀⦇_,RATIFY'⦈_ : RatifyEnv → RatifyState → GovActionID × Gov
 _⊢_⇀⦇_,RATIFY⦈_ : RatifyEnv → RatifyState → List (GovActionID × GovActionState) → RatifyState → Set
 _⊢_⇀⦇_,RATIFY⦈_ = SS⇒BS (λ where (Γ , _) → Γ ⊢_⇀⦇_,RATIFY'⦈_)
 \end{code}
+} %% end small
 \caption{RATIFY transition system}
+\label{fig:ratify-transition-system}
 \end{figure*}
+Finally, Figure~\ref{fig:ratify-transition-system} defines three functions: \AgdaFunction{RATIFY-Accept}, \AgdaFunction{RATIFY-Reject}, and \AgdaFunction{RATIFY-Continue}, along with the syntax \verb!_⊢_⇀⦇_,RATIFY⦈_!.
+The syntax \verb!_⊢_⇀⦇_,RATIFY⦈_! is a wrapper around another function, represented by \AgdaDatatype{SS⇒BS}, which is a transformation from some source state (\AgdaDatatype{SS}) to a binary state (\AgdaDatatype{BS}).
+\begin{itemize}
+  \item \AgdaFunction{RATIFY-Accept} represents an action related to the ratification of a governance action. It takes some context \AgdaDatatype{Γ} a list of actions \AgdaDatatype{es}, and a state \AgdaDatatype{st}
```

Comment:
This explanation doesn't really say what's going on. If an action is `accepted` and not `delayed`, `RATIFY-Accept` enacts it (via `ENACT`) and replaces `EnactState` appropriately. It also removes the action from the queue & sets the `delay` flag properly.